### PR TITLE
[12.0][MIG] purchase_request_usage_department

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

Depends on:

- [x] https://github.com/OCA/purchase-workflow/pull/720

- [x] https://github.com/OCA/purchase-workflow/pull/723

- [x] https://github.com/OCA/purchase-workflow/pull/721

CC @Eficent

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:

**Comment by ps-tubtim** [comment]
LGTM

**Review by ps-tubtim — APPROVED** [review]

File: purchase_request_usage_department/models/hr_department.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (nikul-serpentcs) [inline purchase_request_usage_department/models/hr_department.py]: @aheficent Bro It's Not a `AGPL`, Please Rename as per README.

File: purchase_request_usage_department/tests/test_purchase_request_usage_department.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        dept_dict = {
+            'name': 'department with no usage',
+            'usage_id': self.usage_2.id
+        }
+        self.department_usage_2 = self.dep_model.create(dept_dict)
+        user_dict = {
+            'name': 'User test',
+            'login': 'u@example.com',
+            'email': 'u@example.com',
+            'groups_id': [(4, self.env.ref(
```
Comment (nikul-serpentcs) [inline purchase_request_usage_department/tests/test_purchase_request_usage_department.py]: Here You can Directly get Id Like `self.ref( 'purchase_request.group_purchase_request_user')`

File: purchase_request_usage_department/models/hr_department.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply purchase_request_usage_department/models/hr_department.py]: Noticed. Thanks bro!

**Pull request merged**